### ISSUE TITLE
example doesn’t do what it says

### ISSUE THREAD

**Issue by sydb** [TEIC/Stylesheets#322]
The `@ageAtDeath` example in TD says it shows constraining the attribute to “positive integer values less than 150” but then shows a test that will fail if the value is expressed as a ratio (which teidata.numeric allows), and does not limit the value to positive integers.

That is, the demonstrated definition **does** allow negative integers and positive rational numbers that have a finite number of decimal places; but worse, if handed a rational number expressed as a ration, it is likely to cause the Schematron processor to throw an error.

(I just tried using first probatron and then oXygen. Probatron bombed out when a ratio was supplied, but otherwise correctly reported positive numbers >=150. oXygen never flagged an error, even if the value being tested was 1555.)

A better way to express this, IMHO, would be
~~~~~xml
<datatype minOccurs="1" maxOccurs="1">
  <dataRef name="nonNegativeInteger" restriction="[1-9][0-9]?|1[1-4][0-9]"/>
</datatype>
~~~~~
but that doesn’t exemplify what this section of TD is trying to show.

**Comment by sydb** [comment]
Sub-group (including me) thinks that just changing `teidata.numeric` to `teidata.count` in the example does the job far more simply than my lovely regexp.

**Comment by sydb** [comment]
Fixed at TEI eeb7537.
